cmake/caliper.cmake

```diff
@@ -7,6 +7,8 @@ message(STATUS "Building ${CALIPER}")
 ExternalProject_Add(${CALIPER}
  URL ${CALIPER_URL}/${CALIPER_GZ}
  URL_MD5 ${CALIPER_MD5}
+ PREFIX third-party
+ INSTALL_DIR third-party/install
```

Comment:
I wouldn't installed all of the libraries in the same prefix!

Reply:
I had it as `PREFIX project_name` but then during build legion cannot find gasnet include dir and conduits...